selfdrive/car/tests/test_car_interfaces.py

```diff
@@ -9,11 +11,36 @@
 from selfdrive.car.car_helpers import interfaces
 from selfdrive.car.fingerprints import _FINGERPRINTS as FINGERPRINTS, all_known_cars
 
+def get_random_car_control():
```

Comment:
use the existing stuff and extend it if necessary https://github.com/commaai/openpilot/blob/master/selfdrive/test/process_replay/test_fuzzy.py